modules/adblocker/sources/adblocker.es

```diff
@@ -65,11 +65,27 @@ export default class Adblocker {
     this.whitelist = new UrlWhitelist('adb-blacklist');
     this.stats = new AdbStats();
     this.manager = new EngineManager();
+    this.syncTrustedSites();
+  }
+
+  syncTrustedSites() {
+    this.trustedSites = new Set();
+    for (const site of config.trustedSitesSnapshot) {
+      this.trustedSites.add(site);
+
+      // Drop the leading "www." to be closer to the code in the extension here:
```

Comment:
Nit. Slightly confusing comment here since we seem to be adding `www.` instead of dropping them.

Reply:
Yes, will update it.